src/new/connection.ts

```diff
@@ -31,10 +31,15 @@ class WalletConnection extends BaseConnection {
 
   static CONNECTED: number = 2;
 
+  currentStreamId: string | null = null;
```

Comment:
Why does the connection keep this information? I guess it should belong to the streaming manager (and not the connection itself). Same for the `streamAbortController`.

Reply:
I guess I got a better understanding of what you're doing. Maybe create just one attribute that stores all information about the current streaming and also knows how to abort?

Reply:
Done

Reply:
Thanks!